nixos/tests/flaresolverr.nix

```diff
@@ -0,0 +1,19 @@
+import ./make-test-python.nix (
```

Comment:
is there any more we can do here that would provide value?

Reply:
For now, I don't think so.